### PR TITLE
Modifying Creating CI/CD Pipelines based on revised tutorial

### PR DESCRIPTION
**Pull request by sounix000**
- **Aligned team**: Dev Tools
- **OCP version for cherry-picking**: `4.7`, `4.8`
- **JIRA issues**: [RHDEVDOCS-1860](https://issues.redhat.com/browse/RHDEVDOCS-1860)
- **Preview pages**: [Creating CI/CD solutions for applications using OpenShift Pipelines](https://deploy-preview-31331--osdocs.netlify.app/openshift-enterprise/latest/cicd/pipelines/creating-applications-with-cicd-pipelines.html)
- **Reviewer**: Savita Ashture / Praveen Thangadancha / Pavol Pitonak

### CODE REVIEW

**Review by ppitonak — CHANGES_REQUESTED** [review]
Rename `vote-api` and `vote-ui` also in section `Mirroring images to run pipelines in a restricted environment` (both strings appear twice there)

File: modules/op-creating-pipeline-tasks.adoc [hunk 1]
```diff
@@ -32,7 +32,7 @@ apply-manifests                   1 minute ago
 update-deployment                 48 seconds ago
 ----
 
-. Use the `tkn clustertasks list` command to list the Operator-installed additional cluster tasks, for example, `buildah` and `s2i-python-3`:
+. Use the `tkn clustertasks list` command to list the operator-installed additional cluster tasks such as `buildah` and `s2i-python-3`:
```
Comment (Preeticp) [inline modules/op-creating-pipeline-tasks.adoc:35]: Operator is to stay capital as per OCP guidelines.

File: modules/op-creating-pipeline-tasks.adoc [hunk 2]
```diff
@@ -44,7 +44,7 @@ To use the `buildah` cluster task in a restricted environment, you must ensure t
 $ tkn clustertasks list
 ----
 +
-The output lists the Operator-installed `ClusterTask` resources:
+The output lists the operator-installed `ClusterTask` resources:
```
Comment (Preeticp) [inline modules/op-creating-pipeline-tasks.adoc:47]: Operator

File: modules/op-creating-webhooks.adoc [hunk 3]
```diff
@@ -3,39 +3,39 @@
...
 
 [id="creating-webhooks_{context}"]
-= Creating Webhooks
+= Creating webhooks
 
-_Webhooks_ are HTTP POST messages that are received by the EventListeners whenever a configured event occurs in your repository. The event payload is then mapped to TriggerBindings, and processed by TriggerTemplates. The TriggerTemplates eventually start one or more PipelineRuns, leading to the creation and deployment of Kubernetes resources.
+_Webhooks_ are HTTP POST messages that are received by the event listeners whenever a configured event occurs in your repository. The event payload is then mapped to trigger bindings, and processed by trigger templates. The trigger templates eventually start one or more pipeline runs, leading to the creation and deployment of Kubernetes resources.
 
-In this section, you will configure a Webhook URL on your forked Git repositories `vote-ui` and `vote-api`. This URL points to the publicly accessible EventListener service route.
+In this section, you will configure a webhook URL on your forked Git repositories `pipelines-vote-ui` and `pipelines-vote-api`. This URL points to the publicly accessible EventListener service route.
```
Comment (Preeticp) [inline modules/op-creating-webhooks.adoc:10]: `EventListener`

**Review by Preeticp — COMMENTED** [review]
A few minor nits, otherwise looks good. Thanks for fixing the cases of the CRD instances.

File: modules/op-running-a-pipeline.adoc [hunk 4]
```diff
@@ -17,7 +17,7 @@ A `PipelineRun` resource starts a pipeline and ties it to the Git and image reso
 $ tkn pipeline start build-and-deploy \
     -w name=shared-workspace,volumeClaimTemplateFile=https://raw.githubusercontent.com/openshift/pipelines-tutorial/{pipelines-ver}/01_pipeline/03_persistent_volume_claim.yaml \
     -p deployment-name=vote-api \
```
Comment (ppitonak) [inline modules/op-running-a-pipeline.adoc:19]: this will probably need an update, too

File: modules/op-running-a-pipeline.adoc [hunk 5]
```diff
@@ -32,14 +32,14 @@ $ tkn pipelinerun logs <pipelinerun_id> -f
...
 The <pipelinerun_id> in the above command is the ID for the `PipelineRun` that was returned in the output of the previous command.
 
-. Start the Pipeline for the front-end application:
+. Start the pipeline for the front-end application:
 +
 [source,yaml,subs="attributes+"]
 ----
 $ tkn pipeline start build-and-deploy \
     -w name=shared-workspace,volumeClaimTemplateFile=https://raw.githubusercontent.com/openshift/pipelines-tutorial/{pipelines-ver}/01_pipeline/03_persistent_volume_claim.yaml \
     -p deployment-name=vote-ui \
```
Comment (ppitonak) [inline modules/op-running-a-pipeline.adoc:41]: this will probably need an update, too

**Review by ppitonak — CHANGES_REQUESTED** [review]

File: modules/op-running-a-pipeline.adoc [hunk 6]
```diff
@@ -52,14 +52,14 @@ $ tkn pipelinerun logs <pipelinerun_id> -f
...
 [source,yaml]
 ----
 $ tkn pipelinerun list
 ----
 +
-The output lists the PipelineRuns:
+The output lists the pipeline runs:
 +
 [source,yaml]
 ----
```
Comment (ppitonak) [inline modules/op-running-a-pipeline.adoc:65]: Line 76 also need update

File: modules/op-adding-triggers.adoc [hunk 7]
```diff
@@ -3,9 +3,9 @@
 // *openshift_pipelines/creating-applications-with-cicd-pipelines.adoc
 
 [id="adding-triggers_{context}"]
-= Adding Triggers to a Pipeline
+= Adding triggers to a pipeline
 
-Triggers enable pipelines to respond to external GitHub events, such as push events and pull requests. After you assemble and start a Pipeline for the application, add the `TriggerBinding`, `TriggerTemplate`, `Trigger`, and `EventListener` resources to capture the GitHub events.
+Triggers enable pipelines to respond to external GitHub events, such as push events and pull requests. After you assemble and start a pipeline for the application, add the `TriggerBinding`, `TriggerTemplate`, `Trigger`, and `EventListener` resources to capture the GitHub events.
```
Comment (VeereshAradhya) [inline modules/op-adding-triggers.adoc:9]: small indentation issue in this file.. line number 120
```
apiVersion: triggers.tekton.dev/v1alpha1
kind: Trigger
metadata:
  name: vote-trigger
spec:
  serviceAccountName: pipeline
  bindings:
    - ref: vote-app
  template:
     ref: vote-app
```

**Review by ppitonak — APPROVED** [review]

File: modules/op-adding-triggers.adoc (hunk 7 above)
Comment (ppitonak) [reply modules/op-adding-triggers.adoc:9]: This is the last tiny issue

**Comment by ppitonak** [comment]
All PRs to Tutorial were merged, I tested examples from this doc and everything works. Please update the indentation issue and we are good to go.

File: modules/op-adding-triggers.adoc [hunk 8]
```diff
@@ -63,7 +63,7 @@ spec:
   - apiVersion: tekton.dev/v1beta1
     kind: PipelineRun
     metadata:
-      name: build-deploy-$(tt.params.git-repo-name)-$(uid)
+      generateName: build-deploy-$(tt.params.git-repo-name)-
```
Comment (ppitonak) [inline modules/op-adding-triggers.adoc:66]: Line 134 needs to be updated to `https://github.com/openshift/pipelines-tutorial/raw/{pipelines-ver}/03_triggers/03_trigger.yaml`

**Review by ppitonak — CHANGES_REQUESTED** [review]

**Review by ppitonak — APPROVED** [review]
/lgtm

**Review by ppitonak — APPROVED** [review]
/lgtm

**Pull request merged**